src/disasm.cpp

```diff
@@ -953,543 +1007,543 @@ const BYTE CDetourDis::s_rbModRm[256] = {
     0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0                  // Fx
 };
 
-const CDetourDis::COPYENTRY CDetourDis::s_rceCopyTable[] =
+const BYTE CDetourDis::s_rceCopyTable[] =
 {
-    /* 00 */ ENTRY_CopyBytes2Mod,                      // ADD /r
-    /* 01 */ ENTRY_CopyBytes2Mod,                      // ADD /r
-    /* 02 */ ENTRY_CopyBytes2Mod,                      // ADD /r
-    /* 03 */ ENTRY_CopyBytes2Mod,                      // ADD /r
-    /* 04 */ ENTRY_CopyBytes2,                         // ADD ib
-    /* 05 */ ENTRY_CopyBytes3Or5,                      // ADD iw
+    /* 00 */ eENTRY_CopyBytes2Mod,                      // ADD /r
```

Comment:
You might rephrase the PR with the same intent and even resulting bytes, but without changing these lines. i.e. just to make the diff easier to read.

Reply:
Alternatively, someone else might try to make the same change and verify they get the same result. I understand, you just prepended 'e' to every identifer, over 512 lines, it is just hard to verify.

Reply:
This is what `diff -ur` or `git diff` gave, although less straight. Please let me know how to make this more human readable.

I used a regex to generate the eENTRY_* definitions (value/name pair)  from the original ones.